mods/ra/rules/civilian.yaml

```diff
@@ -70,18 +70,31 @@ FCOM:
 		Dimensions: 2,3
 		LocalCenterOffset: 0,-512,0
 	Valued:
-		Cost: 2000
+		Cost: 1
 	Health:
-		HP: 400
+		HP: 800
 	Armor:
 		Type: Wood
 	Tooltip:
 		Name: Forward Command
+	GivesBounty:
```

Comment:
I think it'd be better to remove this entirely using `-GivesBounty:` instead of setting the `Percentage` to 0.